pydrive2/fs/spec.py

```diff
@@ -534,13 +534,22 @@ def _gdrive_upload_fobj(self, title, parent_id, stream, callback=None):
         return item
 
     def cp_file(self, lpath, rpath, **kwargs):
-        """In-memory streamed copy"""
-        with self.open(lpath) as stream:
-            # IterStream objects doesn't support full-length
-            # seek() calls, so we have to wrap the data with
-            # an external buffer.
-            buffer = io.BytesIO(stream.read())
-            self.upload_fobj(buffer, rpath)
+        """copy a single file or folder"""
+
+        if self.exists(rpath):
```

Comment:
It's an extra API call I think. It's better to rely on the exception that the `file.Copy` would throw